pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -17,30 +17,36 @@ limitations under the License.
 package util
 
 import (
+	fed_client_1_3 "k8s.io/kubernetes/federation/client/clientset_generated/release_1_3"
```

Comment:
@caesarxuchao Just wanted you to take a look at this. Do you think this is the right "new" way to import the clients? Particularly, should the import name have the version as well?